README.md

```diff
@@ -42,7 +42,7 @@ implementation 'co.omise:omise-android:3.0.0'
 
 ## Usage
 
-#### Credit Card activity
+### Credit Card activity
```

Comment:
At some point, we should rewrite this so that it refers to cards, not just credit cards.  I suppose we have to explain it carefully as we do have an class called `CreditCardActivity`.

Reply:
@danfowler Do you some references about the cards in our documents?